std/portable/index.js

```diff
@@ -206,10 +206,18 @@ globalScope["isFloat"] = function isFloat(arg) {
   return typeof arg === "number";
 };
 
+globalScope["isNullable"] = function isNullable(arg) {
+  return arg == null;
+}
+
 globalScope["isReference"] = function isReference(arg) {
   return typeof arg === "object" || typeof arg === "string";
 };
 
+globalScope["isFunction"] = function isFunction(arg) {
+  return typeof arg === "function" || arg instanceof Function;
```

Comment:
Afaik the second case isn't necessary here, as `typeof (new Function()) == "function"`